packages/@aws-cdk/aws-kinesisanalytics-flink/lib/application.ts

```diff
@@ -851,15 +906,24 @@ class Import extends ApplicationBase {
   public readonly applicationName: string;
   public readonly applicationArn: string;
 
-  constructor(scope: Construct, id: string, attrs: { applicationArn: string, applicationName: string }) {
+  constructor(scope: Construct, id: string, attrs: { applicationArn: string, securityGroups?: ec2.ISecurityGroup[] }) {
```

Comment:
why has `applicationName` been removed?

I'd rather avoid removing this arg entirely, as doing so is a breaking change

Reply:
`Import` is an internal class that has never been exported so I don't think it's a breaking change. I think you may have missed that this code is in the `Import` class because the diff view isn't wide enough to show the `class Import` declaration.

Reply:
ahh I see, yep I'd missed that. Thanks